### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.low-contrast-large-aaa)

### ISSUE THREAD

**Issue by joosep41** [joosep41/Manguarvustus#3]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Example Keyboard Trap](file:///C:/Users/j/dev/accessibility-fails/colorcontrast.html)
#### Element path
.low-contrast-large-aaa
#### Snippet
`<p class="low-contrast-large-aaa">This large text does not have enough contrast with it's background</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.7 (foreground color: #81858c, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 109.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.36.1 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
